### DISCUSSION TITLE
Universal Analytics will no longer process new data in standard properties beginning July 1, 2023.

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [freekmurze/laravel-analytics#458] [General]
As Google message shows every time. we want this package supports the Google tag manager or at least to update the docs to the new Google Analytics changes.

![Screen Shot 2022-06-07 at 15 29 01](https://user-images.githubusercontent.com/24838274/172378979-1b9e49fb-d528-4824-b10f-ca8851393d61.png)

thanks.

**Comment by freekmurze** [comment]
Feel free to PR improvements.
